packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -2,13 +2,57 @@ import { html } from '@microsoft/fast-element';
 import { RichTextViewer, richTextViewerTag } from '..';
 import { fixture, type Fixture } from '../../../utilities/tests/fixture';
 import { RichTextViewerPageObject } from '../testing/rich-text-viewer.pageobject';
+import { anchorTag } from '../../../anchor';
+import { waitForUpdatesAsync } from '../../../testing/async-helpers';
+import {
+    RichTextMentionUsers,
+    richTextMentionUsersTag
+} from '../../../rich-text-mention/users';
+import { richTextMentionUsersViewTag } from '../../../rich-text-mention/users/view';
+import { MappingUser, mappingUserTag } from '../../../mapping/user';
 
 async function setup(): Promise<Fixture<RichTextViewer>> {
     return fixture<RichTextViewer>(
-        html`<nimble-rich-text-viewer></nimble-rich-text-viewer>`
+        html`<${richTextViewerTag}></${richTextViewerTag}>`
     );
 }
 
+async function setupMentionConfig(): Promise<Fixture<RichTextViewer>> {
+    return fixture<RichTextViewer>(
+        // prettier-ignore
+        html`<${richTextViewerTag}>
+            <${richTextMentionUsersTag} pattern="^user:(.*)">
+                <${mappingUserTag} key="user:1" display-name="John Doe"></${mappingUserTag}>
+                <${mappingUserTag} key="user:2" display-name="Mary Wilson"></${mappingUserTag}>
+            </${richTextMentionUsersTag}>
+        </${richTextViewerTag}>`
+    );
+}
+
+async function appendUserMentionConfiguration(
```

Comment:
nit: could you make `userKeys` and `displayNames` [optional parameters](https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters) instead of setting their type to include `undefined`? Then the callers wouldn't need to explicitly pass `undefined` for those parameters and could instead call with `appendUserMentionConfiguration(element)`.

Reply:
Made them as optional parameters and removed calling the `undefined` in the function calls within the tests